### DISCUSSION TITLE
if i wanted twitter or facebook, i'd be using them. i want my dashboard on github to be MY stuff, MY repositories. not a feed of other peopel's posts

### DISCUSSION THREAD

**Discussion by crystalwizard** [community/community#64370] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

i find the change to put a spam post feed on my dashboard extremely irritating. i dont' want to see any of that, i want to see only my stuff. I want to be able to set things for my account back to the way they were before someone decided that github needs to look like social media instead of a programming repository site.

**Comment by Mahmood3214** [comment]
Goooooood

**Comment by crystalwizard** [comment]
no one asked you.

On Sun, Aug 20, 2023 at 3:55 AM Mahmood ***@***.***> wrote:

> Goooooood
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/64370#discussioncomment-6772566>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEI3PKTXS64KITCJX2DAEFLXWHUI5ANCNFSM6AAAAAA3XCCI2A>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Visit my store on Teepublic https://www.teepublic.com/user/crystalwizard and
grab something cool today!

**Reply by wendybossons** [reply]
Exactly. That is why it is a problem because the user(s) didn't want it.

**Comment by vncastanheira** [comment]
We need to send a message to Github that we don't want it to be another social media.

**Comment by wendybossons** [comment]
Agreed. What a waste of time trying to find my projects amidst this mess.

**Reply by sarigene2** [reply]
gooo

**Comment by qub1750ul** [comment]
I have always seen GitHub as a code repository with great social/collaborative features, definitely NOT a social network with git mixed somewhere in.
Right now my dashboard is CROWDED with things i don't really care about. I don't come to GitHub for this, never needed this for years before, wouldn't need this any time soon.
If i want to discover software i just search for it, precisely, or refer to awesome lists.
If i want to keep up with a repo or a person i just watch/follow them, and then get updates in the "Following" feed. Coupled with RSS and Email digests this is pretty much what the majority of users will ever need when "social browing" GitHub.
As it stands now the dashboard has a lot of useless additional visual and cognitive burden shoved right into my face.

On a positive note, the "Latest changes" feed is pretty neat and could very well take the place of the "For You" tab.

**Comment by crystalwizard** [comment]
take a hike, Sarigene2. no one asked for your opinion and your snark isn't
appreciated

On Tue, Aug 22, 2023 at 1:44 PM sarigene2 ***@***.***> wrote:

> gooo
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/64370#discussioncomment-6795813>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEI3PKU6KBFSECIGR3SNPCTXWUK2BANCNFSM6AAAAAA3XCCI2A>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Visit my store on Teepublic https://www.teepublic.com/user/crystalwizard and
grab something cool today!
